src/VirtoCommerce.CustomerExportImportModule.Data/Services/CsvCustomerDataValidator.cs

```diff
@@ -24,7 +25,17 @@ public CsvCustomerDataValidator(IBlobStorageProvider blobStorageProvider, ISetti
             _settingsManager = settingsManager;
         }
 
-        public async Task<ImportDataValidationResult> ValidateAsync(string filePath)
+        public async Task<ImportDataValidationResult> ValidateAsync(string dataType, string filePath)
+        {
+            return dataType switch
+            {
+                nameof(Contact) => await ValidateAsync<CsvContact>(filePath),
+                nameof(Organization) => await ValidateAsync<CsvOrganization>(filePath),
+                _ => await ValidateAsync<CsvContact>(filePath),
```

Comment:
I think we should throw exception here instead because we support contacts & organizations only